src/main/java/gregtech/api/capability/impl/MultiblockRecipeLogic.java

```diff
@@ -54,12 +60,29 @@ protected IMultipleTankHandler getOutputTank() {
     protected boolean setupAndConsumeRecipeInputs(Recipe recipe) {
         RecipeMapMultiblockController controller = (RecipeMapMultiblockController) metaTileEntity;
         if (controller.checkRecipe(recipe, false) &&
-            super.setupAndConsumeRecipeInputs(recipe)) {
+            multiBlockSetupAndConsumeRecipeInputs(recipe)) {
             controller.checkRecipe(recipe, true);
             return true;
         } else return false;
     }
 
+    //Logic Mostly copied from AbstractRecipeLogic, but with some additional checking for Multiblock output spacing to prevent voiding
+    protected boolean multiBlockSetupAndConsumeRecipeInputs(Recipe recipe) {
+        int[] resultOverclock = calculateOverclock(recipe.getEUt(), getMaxVoltage(), recipe.getDuration());
+        int totalEUt = resultOverclock[0] * resultOverclock[1];
+        IItemHandlerModifiable importInventory = getInputInventory();
+        IItemHandlerModifiable exportInventory = getOutputInventory();
+        IMultipleTankHandler importFluids = getInputTank();
+        IMultipleTankHandler exportFluids = getOutputTank();
+        List<ItemStack> itemOutputs = recipe.getAllItemOutputs(exportInventory.getSlots());
```

Comment:
This can still void items if for example you use same recipe as before and you have only Export Bus with 1 slot. Then recipe will process but it will gave only Amethyst and void Tiny Pile of Dark Ashes. Which is probably not something we want

Reply:
As mentioned on discord, this is currently not a bug due to the logic of getAllItemOutputs truncating the number of outputs to the maximum number of slots. This should probably be looked at more in depth in a different issue to make sure that all consequences of changing it, if that is what is desired, are handled.

Reply:
Yeah we can skip this one here and follow up on it in #1215